bigtable/tests/unit/test_table.py

```diff
@@ -556,6 +556,12 @@ class Test__RetryableMutateRowsWorker(unittest.TestCase):
     INSTANCE_NAME = ('projects/' + PROJECT_ID + '/instances/' + INSTANCE_ID)
     TABLE_ID = 'table-id'
 
+    # RPC Status Codes
+    SUCCESS = 0
+    RETRYABLE_1 = 4
+    RETRYABLE_2 = 10
+    NON_RETRYABLE = 1
```

Comment:
Please don't hard-code these, i.e. read them from the relevant "source of truth" (if there is one).

Reply:
I replaced the magic numbers with their source values. I'm still keeping the `SUCCESS`, `RETRYABLE_1`, etc. as they are easier to read in the test code with the corresponding comments there.

Reply:
@dhermes @tseaver I noticed that we don't run linter on the test files. Is it just so our CI doesn't take too long (or longer that it already does)?